packages/core/lib/debug/interpreter.js

```diff
@@ -304,10 +293,7 @@ class DebugInterpreter {
     }
 
     //split arguments for commands that want that; split on runs of spaces
-    splitArgs = cmd
-      .trim()
-      .split(/ +/)
-      .slice(1);
+    splitArgs = cmd.trim().split(/ +/).slice(1);
```

Comment:
Note this does not account for a quoted argument with spaces. I know this isn't part of the work done in the PR but wanted to surface it

Reply:
@cds-amal can you make a separate issue for this?

Reply:
note that this is the debug repl, I'm not sure what behavior you're describing @cds-amal, but the use case might be different here.

Reply:
Oh. If there's an expectation for no quoted arguments then all is good. The parsing behavior I tried to explain in the following block.  

```
> cmd = "command arg1 \"arg two\" arg3"
'command arg1 "arg two" arg3'
> cmd.trim().split(/ +/).slice(1)
[ 'arg1', '"arg', 'two"', 'arg3' ]

```

Reply:
Yep, the debugger doesn't really provide that level of sophistication! (Maybe it should? But I think we can ignore for now)